lib/screens/profile/userprofile.dart

```diff
@@ -30,7 +54,11 @@ class _MyProfileState extends State<MyProfile> {
     final currentuser = Provider.of<FirebaseUser>(context);
     //print(currentuser.uid);
 
-    Firestore.instance.collection('userdetails').document(currentuser.uid).get().then((value) {
+    Firestore.instance
+        .collection('userdetails')
+        .document(currentuser.uid)
+        .get()
+        .then((value) {
```

Comment:
Although the formatting seemed to have fixed. But your hardwrap value is not set at 900
**Make sure hard wrap is set to 900 while formatting.**
![](https://user-images.githubusercontent.com/31121102/82408535-67bf2a80-9a89-11ea-80d9-361f72c8b8b4.png)
and set the hardwrap value to 900
If on android studio, select all using CTRL + A and then formatting can be done using CTRL+ALT+L.
If you are on vs code, check this for changing hard wrapping. https://stackoverflow.com/questions/59456452/how-to-change-vs-code-line-auto-wrap-column-when-formatting-dart-files
Check this for more info about formatting. https://flutter.dev/docs/development/tools/formatting